code-style/action.yml

```diff
@@ -49,3 +80,31 @@ runs:
     - name: "Run pre-commit"
       shell: bash
       run: pre-commit run --all-files --show-diff-on-failure
+
+    # ------------------------------------------------------------------------
+
+    # Trigger docker-style action if docker-lint is set to true.
+    - uses: ansys/actions/_logging@main
+      if: ${{ inputs.docker-lint == 'true' }}
```

Comment:
```suggestion
      if: inputs.docker-lint == 'true'
```

Reply:
Not saying that this may not work. Just suggesting the syntax we have in other actions.